deepmd/pt/infer/deep_eval.py

```diff
@@ -624,6 +624,30 @@ def get_model_def_script(self) -> str:
         """Get model definition script."""
         return self.model_def_script
 
+    def get_model_size(self) -> dict:
+        """Get model parameter count.
+
+        Returns
+        -------
+        dict
+            A dictionary containing the number of parameters in the model.
+            The keys are 'descriptor', 'fitting_net', and 'total'.
+        """
+        params_dict = dict(self.dp.named_parameters())
+        sum_param_des = sum(
+            params_dict[k].numel() for k in params_dict.keys() if "descriptor" in k
+        )
+        sum_param_fit = sum(
+            params_dict[k].numel()
+            for k in params_dict.keys()
+            if "fitting" in k and "_networks" not in k
+        )
```

Comment:
If we have model parameters which don't include "descriptor" and "_networks"(maybe in the future), we will neglect them when counting parameters. But I think it is ok now because we don't have this kind of model parameters.

Reply:
Maybe we can add other variants in the future if necessary.